### ISSUE TITLE
Suggestion about library

### ISSUE THREAD

**Issue by niedbalski** [j-kaltes/Juggluco#104]
Hi @j-kaltes great work on this tool.

Question, Is there a chance you can export or guide me to extract the bluetooth/nfc communication library in its own so that we can use it independently?

**Comment by j-kaltes** [comment]
If you want to write an app for Freestyle Libre sensors yourself, you do you need Juggluco?
If you like to receive glucose values or entered numbers out of Juggluco, you can make use of a glucose broadcast (https://www.juggluco.nl/Juggluco/glucosebroadcast.html) or the webserver (https://www.juggluco.nl/Juggluco/webserver.html) in Juggluco.

**Comment by niedbalski** [comment]
> If you want to write an app for Freestyle Libre sensors yourself, you do you need Juggluco? If you like to receive glucose values or entered numbers out of Juggluco, you can make use of a glucose broadcast (https://www.juggluco.nl/Juggluco/glucosebroadcast.html) or the webserver (https://www.juggluco.nl/Juggluco/webserver.html) in Juggluco.

Yes, indeed I could use both. But given you already sorted the Bluetooth and nfc layers communication I was wondering if I could take that work to make a library that can be used in other use cases as well.
